docker/README

```diff
@@ -0,0 +1,18 @@
+# Modifying base image (base.dockerfile)
+
+* Push your changes, create PR and merge it in matser
+* Ask for the rights to https://cloud.docker.com/repository/docker/stitchem/stitchem-base
+* Build the image locally: `docker build --tag stitchem/stitchem-base:latest --file docker/base.dockerfile .`
+* Push the image: `docker push stitchem/stitchem-base:latest`
+
+# Modifying base_cuda_x (base_cuda_x.dockerfile)
+
+* Push your changes, create PR and merge it in matser
+* Ask for the rights to https://cloud.docker.com/repository/docker/stitchem/stitchem-base-cudax
+* Build the image locally: `docker build --tag stitchem/stitchem-base-cudax:latest --file docker/base.dockerfile .`
+* Push the image: `docker push stitchem/stitchem-base-cudax:latest`
+
+# Modifying CI image (ci.dockerfile)
+
+* Push your changes, create PR and merge it in matser
+* You're all set
```

Comment:
Forgive my ignorance, I'm new to Docker and containers. Why is the cloud stuff needed: is that required to extend the basefile or is there some other purpose?

Reply:
The base file are used to reduce the build time, so that we don't have to build the entire container for every build

Reply:
Ah you build the docker images locally and then push them to the Docker Hub, which hosts them for free, as they are public. The Travis CI build doesn't set up any environments, it just pulls the pre-build images from the Docker Hub and then does the CMake commands. Did I get that right?

Reply:
Exactly right. I'll add this in the docker/README.md since it's really not straight forward

Reply:
:+1: